frontends/concrete-python/tests/execution/test_matmul.py

```diff
@@ -44,7 +49,7 @@
         pytest.param(
             (5,),
             (5, 5),
-            (0, 3),
```

Comment:
Let's not remove an existing test, could you add this as a new test?

Reply:
sure! I put it back